core/vm/contracts.go

```diff
@@ -59,6 +59,19 @@ var PrecompiledContractsByzantium = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{8}): &bn256Pairing{},
 }
 
+// PrecompiledContractsConstantinople contains the default set of pre-compiled Ethereum
+// contracts used in the Constantinople release.
+var PrecompiledContractsConstantinople = map[common.Address]PrecompiledContract{
```

Comment:
Dig this, awesome approach IMO. Only thing I wonder is if we should refer to e.g. `PrecompiledContractsByzantium[[]byte{1}]` in cases where the value is the same, to make it immediately obvious that the value is the same.